versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -105,16 +108,26 @@ public long expireAfterRead(
         .ifPresent(
             meterRegistry -> {
               cacheBuilder.recordStats(() -> new CaffeineStatsCounter(meterRegistry, CACHE_NAME));
-              meterRegistry.gauge(
-                  "cache_capacity_mb",
-                  singletonList(Tag.of("cache", CACHE_NAME)),
-                  "",
-                  x -> config.capacityMb());
+              List<Tag> tags = singletonList(Tag.of("cache", CACHE_NAME));
+              meterRegistry.gauge("cache_capacity_mb", tags, "", x -> config.capacityMb());
+              meterRegistry.gauge("cache_weight_mb", tags, "", x -> weightedSizeMb());
             });
 
     this.cache = cacheBuilder.build();
   }
 
+  private double weightedSizeMb() {
+    // Note: calling Policy.Eviction::weightedSize may actually perform evictions.
```

Comment:
I suppose evictions in this call may only happen in practice under load, which may